tests/core/test_scraper.py

```diff
@@ -168,6 +169,15 @@ def test_get_ethpm_birth_block(w3, interval):
     assert actual == latest_block.number - 1
 
 
+def test_get_ethpm_birth_block_invalid_range_raises_exception(w3):
+    latest_block = w3.eth.getBlock("latest")
+    with pytest.raises(BlockNotFoundError):
+        get_ethpm_birth_block(w3, 0, w3.eth.blockNumber, latest_block.timestamp)
+    with pytest.raises(BlockNotFoundError):
+        time_travel(w3, 1)
+        get_ethpm_birth_block(w3, w3.eth.blockNumber, 0, latest_block.timestamp)
```

Comment:
For when the `to_block` is strictly larger than the `from_block`

Reply:
cool! notes like this are definitely worth a comment in the code - and honestly it might be better off as a separate test - then in the test name you can explain what's going on

Reply:
Sure, still trying to discern when it's bloat and when it's appropriate for clarity. Any rule of thumbs you use?

Reply:
Hmmm, imo with tests bloat is not such a big concern, I tend to default to the more tests the better. It's good to have only one specific cause/effect being tested per test. We like to use `@pytest.mark.parametrize` ([example](https://github.com/ethpm/ethpm-cli/blob/master/tests/core/test_config.py#L63)) a lot which helps improve the testing quality while reducing bloat.  ping @pipermerriam for thoughts